app/models/saved_recipie.rb

```diff
@@ -0,0 +1,10 @@
+class SavedRecipie < ApplicationRecord
+  ## VALIDATIONS
+  validates :api_recipie_id, numericality: { only_integer: true }
+  validates :favorited, inclusion: [true, false]
+  validates_associated :user
```

Comment:
🤯 I took some time to look into this validation vs. what you originally had yesterday, `validates :user, presence: true` and found it very interesting. I think this validation is great for now, but we might want to consider the pros and cons of each in the future. I kinda like that the `validates :user, presence: true` option checks that the association is valid but does not run all the validations on that associated object. It seems like  `validates_associated` might add unnecessary validation checks by running all `user` validations and all `saved_recipie` validations. 🚙 Just some food for thought down the road on this project...
🤩 Thanks for showing me the other validation so that I could dig in and explore that further!!

Reply:
I'm going to keep the `validates_associated :user` for now as adding `validates :user, presence: true` causes rubocop to fail becuase of `,presence: true` but when I remove it RSpec fails. However both pass with the original validation

Reply:
I did not expect you to change it. I was just commenting/noting what I found after looking into the two options. But its also is good to know that Rubocop does not like it the other way.